models/icatcher-semireal/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "download": {
+    "file": "https://civitai.com/api/download/models/556730",
+    "autoencoder": ""
+  },
+  "name": "iCatcher Semireal",
+  "file": "icatcher_semireal_f16.ckpt",
+  "prefix": "",
+  "modifier": "none",
+  "default_scale": 16,
+  "upcast_attention": false,
+  "version": "sdxl_base_v0.9",
+  "autoencoder": "sdxl_vae_v1.0_f16.ckpt",
+  "clip_encoder": "icatcher_semireal_sdxl_clip_vit_l14_f16.ckpt",
```

Comment:
remove this and the one below. These will be auto-filled in when the model is converted. It is only useful if your fine-tune uses vanilla text encoders then you reference the vanilla one here.